src/elements/tiles/Tiles.scss

```diff
@@ -16,6 +16,10 @@ novo-tiles {
         cursor: pointer;
         align-items: center;
 
+        &.disabled {
+            opacity: .5;
```

Comment:
Okay I added the toast logic to this class to hopefully make this more clear. If disabled, everything is disabled except toasts if toastOptions are passed in. Do you think the var name still needs to be changed?